app/grandchallenge/evaluation/models.py

```diff
@@ -1864,14 +1869,20 @@ def inputs_complete(self):
         else:
             return False
 
-    @cached_property
-    def additional_inputs_complete(self):
+    @property
+    def additional_inputs(self):
+        # additional inputs as currently defined on the phase
         phase_input_slugs = self.submission.phase.inputs.values_list(
             "slug", flat=True
         )
-        return self.inputs.filter(
-            interface__slug__in=phase_input_slugs
-        ).count() == len(phase_input_slugs)
+        return self.inputs.filter(interface__slug__in=phase_input_slugs)
```

Comment:
Would this work if an additional input is also an algorithm output? Maybe we need to say that the inputs and algorithm outputs must be disjoint. I can't think of a situation where that would be desired but you never know what users will come up with.

Reply:
Ah that's a cornercase where this would break indeed. I think it's a very sensible restriction to add. I'll work on adding validation to the admin form.

Reply:
I'll make a seperate PR for that restriction. I'm wondering if just checking this in the admin form is enough, or if I also need to add a check to the AlgorithmInterfaceForm (in case someone first configures the inputs and then the algorithm interfaces). Adding a check to the AlgorithmInterfaceForm will be a bit trickier I think.

Reply:
I guess it needs to be checked in every place that these two fields could change.

Reply:
I agree. I'll work on that probably tomorrow, and make a seperate PR. The places that will need to have a check are the PhaseAdmin and the AlgorithmInterfaceForm.